lib/listeners/statful.js

```diff
@@ -29,7 +19,7 @@ function dispatchMetricLine(listener) {
             });
 
             if (listener.config.stats) {
-                listener.statfulClient.counter('metrics_flushed', countMetricsInLines(lines));
+                listener.statfulClient.counter('metrics_flushed', linesList.length);
```

Comment:
We want to know the number of metrics flushed here - this way we're only measuring the length of the payload string.